.github/workflows/ci.yml

```
@@ -31,7 +31,9 @@ jobs:
     # See: https://github.com/gradle/actions/blob/main/setup-gradle/README.md
     - name: Setup Gradle
       uses: gradle/actions/setup-gradle@v4.0.0
-
+    # Setup for misc tests
+    - name: Install shellcheck and checkbashisms
+      run: sudo apt install shellcheck devscripts
```

```suggestion
      run: sudo apt install shellcheck devscripts
      if: matrix.script == 'misc'
```

Avoid the extra work for all the other tests.